apps/native-component-list/webpack.config.js

```diff
@@ -226,29 +221,31 @@ module.exports = {
     }),
 
     new webpack.DefinePlugin(env),
-    ...useWebModules(['Platform', 'DeviceInfo', 'Dimensions', 'Linking', 'Image', 'Share', 'Text']),
-    ...useWebModules(
-      [
-        'Performance/Systrace',
-        ['HMRLoadingView', 'Utilities/HMRLoadingView'],
-        ['RCTNetworking', 'Network/RCTNetworking'],
-      ],
-      'expo/build/web/'
-    ),
+
+    useWebModule('Platform', 'Utilities/Platform'),
```

Comment:
Does this override both react-native and expo-core's Platform?

Reply:
This takes any instance of Platform used internally in react-native and redirects it to use the expo/web version

Reply:
What do you think about adding `select()` to expo-core's Platform and then having web/utilities/Platform export that? That way we have a consistent Platform everywhere.

Reply:
Platform has a couple more features. In order to really fix this we'd need to PR a web version into RN, or pull out `Platform` into another library that has web support.

Reply:
OK, it seems like something worth doing eventually